pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -0,0 +1,122 @@
+    <TargetFrameworkName>netcoreapp</TargetFrameworkName>
+    <TargetFrameworkVersion>1.1</TargetFrameworkVersion>
+    <TargetFramework>$(TargetFrameworkName)$(TargetFrameworkVersion)</TargetFramework>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <!-- don't build if we're missing files, we'll intentionally add a missing file if native build is missing -->
+    <SkipCreatePackageOnMissingFiles>true</SkipCreatePackageOnMissingFiles>
+
+    <RefBinDir>$(BinDir)$(TargetFrameworkName)/pkg/ref</RefBinDir>
+    <LibBinDir>$(BinDir)$(TargetFrameworkName)/pkg/lib</LibBinDir>
+    <NativeBinDir>$(BinDir)$(OSGroup).$(PackagePlatform).$(ConfigurationGroup)/native</NativeBinDir>
+
+    <NETStandardLibraryPackage>NETStandard.Library2</NETStandardLibraryPackage>
+    <NETStandardLibraryPackageVersion>2.0.0-beta-24709-0</NETStandardLibraryPackageVersion>
+    <NETStandardVersion>1.7</NETStandardVersion>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <SuppressNETStandardMissingFile Include="mscorlib" />
+    <SuppressNETStandardMissingFile Include="netstandard" />
+    <SuppressNETStandardMissingFile Include="System.Core" />
+    <SuppressNETStandardMissingFile Include="System" />
+    <SuppressNETStandardMissingFile Include="System.Drawing" />
+    <SuppressNETStandardMissingFile Include="System.Numerics" />
+    <SuppressNETStandardMissingFile Include="System.Runtime.Serialization" />
+    <SuppressNETStandardMissingFile Include="System.Web" />
+    <SuppressNETStandardMissingFile Include="System.Xml" />
+    <SuppressNETStandardMissingFile Include="System.Xml.Linq" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
+    <ProjectReference Include="@(Project)" />
+    
+    <!-- Include refs -->
+    <File Include="$(RefBinDir)/*.*">
+      <TargetPath>ref/$(TargetFramework)</TargetPath>
+    </File>
+
+    <FilePackageDependency Include="Microsoft.NETCore.Platforms">
+      <Version>$(PlatformPackageVersion)</Version>
+    </FilePackageDependency>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
+    <!-- Include lib -->
+    <File Include="$(LibBinDir)/*.*">
+      <TargetPath>runtimes/$(PackageTargetRuntime)/lib/$(TargetFramework)</TargetPath>
+    </File>
+
+    <!-- Include native -->
+    <ExcludeNative Include="$(NativeBinDir)/*.lib" />
+    <NativeFile Include="$(NativeBinDir)/*.*" Exclude="@(ExcludeNative)">
+      <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
+    </NativeFile>
+    <File Include="@(NativeFile)" />
+    <!-- force a missing file if native build is absent -->
+    <File Include="$(NativeBinDir)/MISSING_NATIVE_BUILD" Condition="'@(NativeFile)' == ''" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <IgnoredReference Include="System.Private.CoreLib" />
+  </ItemGroup>
+
+  <Target Name="VerifyClosure" AfterTargets="Build">
```

Comment:
Currently this is verifying the closure of the netcoreapp package did you plan to do the same for the full runtime output directory at some point?